code/src/components/app-dropdown/components/AppDropdownOptions.tsx

```diff
@@ -0,0 +1,28 @@
+import { MouseEvent } from "react";
+import AppBox from "@/components/app-box/AppBox";
+import { AppDropdownOptionsProps } from "@/components/app-dropdown/AppDropdown.types";
+
+import "@/components/app-dropdown/AppDropdown.scss";
+
+const AppDropdownOptions = ({
+  options,
+  handleOptionClick
+}: AppDropdownOptionsProps) => {
+  return (
+    <AppBox className="app-dropdown__options">
+      {options.map((option) => (
+        <AppBox
+          key={option.value}
+          className="app-dropdown__option"
+          onClick={(event: MouseEvent) =>
+            handleOptionClick(event, option.value, option.label)

```

Comment:
What do you think about moving this logic above return ?

Reply:
> What do you think about moving this logic above return ?

I created this separate component only to render the items with `map`.
There are not reason to move it above